### PR TITLE
[DOC] Fix example and typo

### LINKED ISSUE #123053: [docs] nn.ChannelShuffle example code unrealistic (`randn` cannot produce integers like `15`)
### 📚 The doc issue

Originally reported in https://github.com/pytorch/pytorch/pull/123033#issuecomment-2028444096

---

The online docs assume any-dimension `(*, C, H, W)`, right?

Also, the code example in online docs seems incorrect as `torch.randn` should return float values around 0 (including negatives) and not integers like `15`:
image
 

To make the example correct, maybe torch.randn should just be replaced by initting `input` by the input tensor from the example

Also, it would be great if the docs example printed the input.shape and output.shape, since it appears that ChannelsShuffle is introducing a new dimension (?): 

strange below-the-line comma:
image

It would be nice if somehow docs code examples were testable (maybe worth somewhere storing test "input"/"random" tensors to not depend on un-reproducible random inputs)

### Suggest a potential alternative/fix

_No response_

cc @svekars @brycebortree @albanD @mruberry @jbschlosser @walterddr @mikaylagawarecki

### LINKED ISSUE #123554: The PRELU formula issue
### 📚 The doc issue

https://pytorch.org/docs/stable/generated/torch.nn.PReLU.html#prelu
The doc shows that the prelu formula is: y = x >=0 ? x : a*x  (1)
but according to the paper "Delving Deep into Rectifiers:Surpassing Human-Level Performance on ImageNet Classification",
the formula is: y = x >0 ? x : a*x (2)
There are big difference when a is special number such as NaN or INF and x is 0, according to the formula (1), y = 0,
but according to the formula (2), y = NaN

### Suggest a potential alternative/fix

Change the doc's formula to "y = x >0 ? x : a*x"

### PR DESCRIPTION
**Pull request by arunppsg**
Fixes #123554 and fixes #123053

### CODE REVIEW

File: torch/nn/modules/channelshuffle.py [hunk 1]
```diff
@@ -9,38 +9,35 @@ class ChannelShuffle(Module):
     r"""Divides and rearranges the channels in a tensor.
 
     This operation divides the channels in a tensor of shape :math:`(*, C , H, W)`
-    into g groups and rearranges them as :math:`(*, \frac{C}{g}, g, H, W)`,
-    while keeping the original tensor shape.
+    into g groups as :math:`(*, C \frac g, g, H, W)` and rearranges them,
```
Comment (mikaylagawarecki) [inline torch/nn/modules/channelshuffle.py:12]: The rendering here seems wrong

https://docs-preview.pytorch.org/pytorch/pytorch/123959/generated/torch.nn.ChannelShuffle.html#channelshuffle
Screenshot 2024-04-12 at 5 39 26 PM

Comment (arunppsg) [reply torch/nn/modules/channelshuffle.py]: Fixed it in [8abfdf8](https://github.com/pytorch/pytorch/pull/123959/commits/8abfdf8120411f67aa2375af30b45e807baab870) and it seems to render correct locally now:
Screenshot 2024-04-14 at 4 19 06 PM

**Review by mikaylagawarecki — APPROVED** [review]
Thanks

**Pull request closed without merging**